staging/src/k8s.io/client-go/util/cert/cert_test.go

```diff
@@ -0,0 +1,30 @@
+package cert_test
```

Comment:
thanks, this file needs the copyright header added

Reply:
done! sorry about that.